lib/tasks/emailer.rake

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+namespace :emailer do
+  desc 'email task runner'
+  task vouchers_to_merchants: :environment do
+    desc 'sends all voucher codes to all of their merchants'
```

Comment:
currently it sends every gift card they have — i wonder if that's what they want, or do they actually just want the ones within the last week

we should ask chloe and marcia what they think the merchants would prefer. it's worth mentioning that sending them everything would include updated values

Reply:
Gotchya, I was imagining we would send them any gift card updated within the last X number of days/weeks, but I suppose maybe it's easier to send them a list of all gift cards every time and they can just toss the last list?

Would be good to hear what they might prefer

Reply:
yeah it's important to note that the values on our dbs are the customer's values, while the values on their paper is the true value — so they'd have to "merge" the lists every time if they did that. we def need seller input here